src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -163,9 +157,6 @@ class SpoGroupMemberAddCommand extends SpoCommand {
       };
 
       const sharingResult = await request.post<SharingResult>(requestOptions);
-      if (sharingResult.ErrorMessage !== null) {
-        throw sharingResult.ErrorMessage;
-      }
```

Comment:
Didn't see the point of this since we have a catch just a few lines below this, which handles the error

Reply:
Just did some tests on the API endpoint. When an error happens, the API returns a 200 request with just an error object in it. That's why this was made I guess, because according to our request package, the API request did not fail so no exception is thrown.

Reply:
In that case we cannot remove it, thank you!